index.js

```diff
@@ -234,7 +235,7 @@ export default class SwitchSelector extends Component {
                         }
                       ],
                       borderRadius: borderRadius,
-                      marginTop: hasPadding ? valuePadding : 0
+                      margin: buttonMargin ? buttonMargin : 0
```

Comment:
After adding buttonMargin to defaultProps with value 0:
```suggestion
                      margin: buttonMargin,
```